pelican/__init__.py

```diff
@@ -500,7 +501,8 @@ def listen(server, port, output, excqueue=None):
 def main(argv=None):
     args = parse_arguments(argv)
     logs_dedup_min_level = getattr(logging, args.logs_dedup_min_level)
-    init_logging(args.verbosity, args.fatal,
+    init_logging(level=args.verbosity, fatal=args.fatal,
+                 handler=RichHandler(console=console), name=__name__,
```

Comment:
Pass the same *console* here so the logging and spinners play nice together.

Reply:
I'd prefer if this was handled in the `log.py` (as part of default `init_logging`) and `__init__.py` would import the `console` from `log.py`.

Reply:
done!